src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1613,7 +1636,10 @@ def _execute_command(self, **kwds):
 
     def __call__(self, **kwds):
         """Call a command with the specified keyword arguments."""
-        return self.execute_command(**kwds)
+        try:
+            return self.execute_command(**kwds)
+        except KeyboardInterrupt:
+            self._interrupt()
```

Comment:
Here we shall check whether the command is interruptible. We won't need to try..except within the `_interrupt()` method.